Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml

```diff
@@ -497,4 +497,15 @@
     - type: Seed
       seedId: bungo
     - type: Sprite
-      sprite: Objects/Specific/Hydroponics/bungo.rsi
\ No newline at end of file
+      sprite: Objects/Specific/Hydroponics/bungo.rsi
+
+- type: entity
```

Comment:
Please double check your [YAML conventions](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions)

Reply:
I have changed the order to be parent, id, name and then description.